src/main/java/mate/academy/internetshop/controllers/ProductDeleteController.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.service.ProductService;
+
+public class ProductDeleteController extends HttpServlet {
```

Comment:
```suggestion
public class DeleteProductController extends HttpServlet {
```
lets write due to English rules

Reply:
Done